packages/backend-dynamic-feature-service/src/server/plugin.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  coreServices,
+  createBackendPlugin,
+} from '@backstage/backend-plugin-api';
+import { createRouter } from './router';
+import { dynamicPluginsServiceRef } from '@backstage/backend-dynamic-feature-service';
+
+/**
+ * frontendRemotesServerPlugin backend plugin
+ *
+ * @internal
+ */
+export const frontendRemotesServerPlugin = createBackendPlugin({
+  pluginId: 'core.dynamicplugins.frontendRemotes',
+  register(env) {
+    env.registerInit({
+      deps: {
+        logger: coreServices.logger,
+        auth: coreServices.auth,
+        httpAuth: coreServices.httpAuth,
+        httpRouter: coreServices.httpRouter,
+        discovery: coreServices.discovery,
+        metadata: coreServices.pluginMetadata,
+        dynamicPlugins: dynamicPluginsServiceRef,
+      },
+      async init({ logger, httpRouter, discovery, metadata, dynamicPlugins }) {
+        httpRouter.use(
+          await createRouter({
+            logger,
+            discovery,
+            metadata,
+            dynamicPlugins,
+          }),
+        );
+        httpRouter.addAuthPolicy({
+          path: `/`,
+          allow: 'unauthenticated',
```

Comment:
Is this really needed? Do we want or need them unauthed? Can we use cookie auth like techdocs has instead?

Reply:
I didn't require this anymore when using a service that adds a router to the rootHttpRouter.
Did I miss something ?